.github/workflows/post-commit.yml

```diff
@@ -346,6 +346,10 @@ jobs:
             maven-repo-${{ hashFiles('**/pom.xml') }}-8
             maven-repo-${{ hashFiles('**/pom.xml') }}
             maven-repo-
+      - name: Setup java
+        uses: actions/setup-java@v1
```

Comment:
Question: Looks like there are newer versions with better support for other JDKs and there is also https://github.com/oracle-actions/setup-java

Reply:
We can check upgrade to v2 or v3 separately.  I don't know if using oracle-actions is OK or not for Apache.

Reply:
We can switch to [setup-java@v3](https://github.com/actions/setup-java#basic) later. And the `temurin` distribution seems more neutral.